addons/web/tests/test_click_everywhere.py

```diff
@@ -7,11 +7,19 @@
 class TestMenusAdmin(odoo.tests.HttpCase):
 
     def test_01_click_everywhere_as_admin(self):
-        self.browser_js("/web", "odoo.__DEBUG__.services['web.clickEverywhere']();", "odoo.isReady === true", login="admin", timeout=45*60)
+        menus = self.env['ir.ui.menu'].load_menus(False)
+        for app in menus['children']:
+                with self.subTest(app=app['name']):
```

Comment:
What is the usage of app parameter ?
Maybe you want `msg = app['name']`
I would prefer a logger at the beginning of the subtest in order to now what app is being tested when reading logs.